### PR TITLE
Description of Plotting Backends added in the end.

### PR DESCRIPTION
**Pull request by mehersaipreetam**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -1641,3 +1641,32 @@ when plotting a large number of points.
...
+
+
+Plotting Backend
+--------------------------------
+Starting in 0.25 pandas can be extended with third-party plotting backends.
+The main idea is letting users select a plotting backend different than the provided one based on Matplotlib.
+For example:
+.. ipython:: python
+    pd.set_option('plotting.backend', 'backend.module')
+    pd.Series([1, 2, 3]).plot()
```
Comment (MarcoGorelli) [inline doc/source/user_guide/visualization.rst]: Does this render correctly? I think you'll need some blank lines, e.g.

```
For example:

.. ipython:: python

    pd.set_option('plotting.backend', 'backend.module')
```

To check whether the page builds correctly, you can do (once you've navigated to the `doc` directory):
```
python make.py clean
python make.py --single user_guide/visualization.rst
```

Comment (mehersaipreetam) [reply doc/source/user_guide/visualization.rst]: Thank You so much for the inputs. I'll review my submission and try again.

Comment (mehersaipreetam) [reply doc/source/user_guide/visualization.rst]: Can you please help me? It says there's a syntax error when I try the clean command in line 63 make.py. I have all the dependencies installed. Where am I going wrong?

Comment (mehersaipreetam) [reply doc/source/user_guide/visualization.rst]: Please help me out. I have the package installed.
Run ! grep -B1 "^<<<-------------------------------------------------------------------------$" sphinx.log
ValueError: Could not find plotting backend 'backend.module'. Ensure that you've installed the package providing the 'backend.module' entrypoint, or that the package has a top-level `.plot` method.
<<<-------------------------------------------------------------------------
--
ModuleNotFoundError: No module named 'backend'
<<<-------------------------------------------------------------------------
--
NameError: name 'backend' is not defined
<<<-------------------------------------------------------------------------
##[error]Process completed with exit code 1.

**Comment by WillAyd** [comment]
@mehersaipreetam is this still active? Can you fix up merge conflicts and remove the added file?

**Pull request closed without merging**